src/hotspot/cpu/x86/x86.ad

```diff
@@ -1681,6 +1681,15 @@ const bool Matcher::match_rule_supported_vector(int opcode, int vlen, BasicType
         return false;
       }
       break;
+    case Op_CopySignD:
+    case Op_CopySignF:
+      if (UseAVX < 3)  {
+        return false;
+      }
+      if (!VM_Version::supports_avx512vl()) {
+        return false;
+      }
+      break;
```

Comment:
This should be part of match_rule_supported() method and not match_rule_supported_vector().
Also should return false for non LP_64.